src/components/Thumbnail/Thumbnail.scss

```diff
@@ -13,15 +8,12 @@ $stacking-order: (
   min-width: thumbnail-size(small);
   max-width: 100%;
   border-radius: var(--p-border-radius-base, border-radius());
+  box-shadow: inset 0 0 0 1px rgba(33, 43, 54, 0.1);
```

Comment:
Can we use an existing shadow from tokens here?

Reply:
I have replaced this with `border('divider')` I can't see any reason it should be an inset line.